source/_patterns/_animations.scss

```diff
@@ -86,36 +86,6 @@ $light-attention-color: $color-srf-red-500;
 
 
 
-/*
- *
- * Ripple-Effect
- * Used by _ripple.scss
- *
- */
-
-@keyframes rippleShadowAnimation {
-  0% {
-    box-shadow: 0 0 0 $color-srf-neutral-blacka00;
-  }
-
-  20% {
-    box-shadow: 0 4px 16px $color-srf-neutral-blacka30;
-  }
-
-  100% {
-    box-shadow: 0 0 0 $color-srf-neutral-blacka00;
-  }
-}
-
-@keyframes rippleAnimation {
-  to {
-    transform: scale(24);
-    opacity: 0;
-  }
-}
-
-
-
 /*
```

Comment:
das Ge-Ripple löschen wär streng formal in einem eigenen PR, ich weiss da nix, aber ok für mich.